moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -39,7 +39,7 @@
 
 #include <geometry_msgs/msg/pose_stamped.hpp>
 #include <moveit/moveit_cpp/moveit_cpp.h>
-#include <moveit/moveit_cpp/plan_solutions.hpp>
+#include <moveit/moveit_cpp/plan_solutions.h>
```

Comment:
Pertaining to the previous file, is this okay?

Reply:
Relates to the above I will switch back to .hpp

Reply:
[5578fde](https://github.com/ros-planning/moveit2/pull/1546/commits/5578fde8e758d827d0c08145b28b51844d07a59f)